docs-client/src/index.css

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+pre, code, kbd, var, samp, tt, .code {
+  font-family: 'Roboto Mono', 'Consolas', 'Menlo', 'Monaco', 'Lucida Console', 'Liberation Mono',
+    'DejaVu Sans Mono', monospace !important;
```

Comment:
Could you avoid using `!important`?

Reply:
Since we're using material-ui components for the majority of our styling, and it defines it's own styles, it can be tricky to override fonts - the styles for each component are local to the component, so generally you would define stylized versions of each component you want to override. I thought it's probably overkill for this one single easy-to-grok CSS rule but can play with it some more if it's still a concern.

Reply:
Does this solution work? https://stackoverflow.com/a/45403932

Reply:
It applies the font to all typography, but this one is only for code snippets. By the way, the default font already uses `Roboto` per material-ui convention, so I didn't change the default typography.

Reply:
Ah I see. Let's leave this as it is for now.